### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/DrizzlePac/align_multiple_visits/align_multiple_visits.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#183]
## What
An accessibility scan flagged 31 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/DrizzlePac/align_multiple_visits/align_multiple_visits.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<pre id="codecell3">` (selector: `#codecell3`)
- `<pre id="codecell4"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9irw3fwq_flc.fits to ./mastDownload/HST/j9irw3fwq/j9irw3fwq_flc.fits ...
</pre>` (selector: `#codecell4`)
- `<pre id="codecell6"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9irw4b1q_flc.fits to ./mastDownload/HST/j9irw4b1q/j9irw4b1q_flc.fits ...
</pre>` (selector: `#codecell6`)
- `<pre id="codecell8"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9irw4b3q_flc.fits to ./mastDownload/HST/j9irw4b3q/j9irw4b3q_flc.fits ...
</pre>` (selector: `#codecell8`)
- `<pre id="codecell10"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9irw5kaq_flc.fits to ./mastDownload/HST/j9irw5kaq/j9irw5kaq_flc.fits ...
</pre>` (selector: `#codecell10`)
- `<pre id="codecell15">` (selector: `#codecell15`)
- `<table id="table140377612372368" class="table-striped table-bordered table-condensed">` (selector: `#table140377612372368`)
- `<pre id="codecell16">` (selector: `#codecell16`)
- `<table id="table140377655468560" class="table-striped table-bordered table-condensed">` (selector: `#inspect-the-alignment > .cell.container.docutils > .cell_output.container.docutils > .text_html.output > div > table`)
- `<pre id="codecell17">` (selector: `#codecell17`)
- `<pre id="codecell18">` (selector: `#codecell18`)
- `<pre id="codecell20">` (selector: `#codecell20`)
- `<pre id="codecell28"><span></span>Python Version 3.11.14 | packaged by conda-forge | (main, Oct 22 2025, 22:46:25) [GCC 14.3.0]
</pre>` (selector: `#codecell28`)
- `<pre id="codecell40"><span></span>  #  Source finding for &apos;j9irw3fwq_flc.fits&apos;, EXT=(&apos;SCI&apos;, 1) started at: 19:07:19.708 (23/10/2025)
</pre>` (selector: `#codecell40`)
- `<pre id="codecell42"><span></span>  #  Source finding for &apos;j9irw3fwq_flc.fits&apos;, EXT=(&apos;SCI&apos;, 2) started at: 19:07:20.140 (23/10/2025)
</pre>` (selector: `#codecell42`)
- `<pre id="codecell47"><span></span>  #  Source finding for &apos;j9irw4b1q_flc.fits&apos;, EXT=(&apos;SCI&apos;, 1) started at: 19:07:20.726 (23/10/2025)
</pre>` (selector: `#codecell47`)
- `<pre id="codecell49"><span></span>  #  Source finding for &apos;j9irw4b1q_flc.fits&apos;, EXT=(&apos;SCI&apos;, 2) started at: 19:07:21.070 (23/10/2025)
</pre>` (selector: `#codecell49`)
- `<pre id="codecell54"><span></span>  #  Source finding for &apos;j9irw5kaq_flc.fits&apos;, EXT=(&apos;SCI&apos;, 1) started at: 19:07:21.707 (23/10/2025)
</pre>` (selector: `#codecell54`)
- `<pre id="codecell56"><span></span>  #  Source finding for &apos;j9irw5kaq_flc.fits&apos;, EXT=(&apos;SCI&apos;, 2) started at: 19:07:22.046 (23/10/2025)
</pre>` (selector: `#codecell56`)
- `<pre id="codecell70"><span></span>Matching sources from &apos;j9irw4b1q_flc.fits&apos; with sources from reference image &apos;j9irw3fwq_flc.fits&apos;
</pre>` (selector: `#codecell70`)
- `<pre id="codecell72"><span></span>Found initial X and Y shifts of -0.001445, -0.00289 with significance of 668.5 and 692 matches
</pre>` (selector: `#codecell72`)
- `<pre id="codecell91"><span></span>Matching sources from &apos;j9irw5kaq_flc.fits&apos; with sources from reference image &apos;j9irw3fwq_flc.fits&apos;
</pre>` (selector: `#codecell91`)
- `<pre id="codecell93"><span></span>Found initial X and Y shifts of 0, 0.002805 with significance of 690.5 and 713 matches
</pre>` (selector: `#codecell93`)
- `<pre id="codecell110">` (selector: `#codecell110`)
- `<pre id="codecell112">` (selector: `#codecell112`)
- `<table id="table140377655468560" class="table-striped table-bordered table-condensed">` (selector: `.cell.container.docutils:nth-child(6) > .cell_output.container.docutils > .text_html.output > div > table`)
- `<pre id="codecell121"><span></span>j9irw4b1q  SCALE_X: 0.9999720117  SCALE_Y: 1.000014811  ROT_X: 359.9999  ROT_Y:   0.0014   SKEW: -359.9985
j9irw5kaq  SCALE_X: 0.9999851345  SCALE_Y: 1.000030317  ROT_X:   0.0015  ROT_Y: 359.9989   SKEW:  359.9974
</pre>` (selector: `#codecell121`)
- `<pre id="codecell122">` (selector: `#codecell122`)
- `<pre id="codecell126">` (selector: `#codecell126`)
- `<pre id="codecell128">` (selector: `#codecell128`)
- `<pre id="codecell132">` (selector: `#codecell132`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
